### PR TITLE
Set versioning information.

### PR DESCRIPTION
**Pull request by ghisvail**
Whilst preparing a test package for `forge`, I noticed the versioning information were missing for the DSO. These settings currently set the soversion to 0, assuming the API is not stable yet. Feel free to change it to a different value before the first stable release.

### CODE REVIEW

File: src/backend/opengl/CMakeLists.txt [hunk 1]
```diff
@@ -208,6 +208,10 @@ ELSE(USE_SYSTEM_GLBINDING)
     ADD_DEPENDENCIES(forge glbinding)
 ENDIF(USE_SYSTEM_GLBINDING)
 
+SET_TARGET_PROPERTIES(forge PROPERTIES
+                      VERSION ${FG_VERSION}
+                      SOVERSION 0)
```
Comment (pavanky) [inline src/backend/opengl/CMakeLists.txt]: We are using `AF_VERSION_MAJOR` for `SOVERSION` in arrayfire. Can you use `FG_VERSION_MAJOR` here for consistency ?

**Comment by shehzan10** [comment]
As it stands, `FG_VERSION` is going to be 1.0.beta which would probably be invalid. So as @pavanky suggested, I think we should go with major version.

**Comment by pavanky** [comment]
@shehzan10 having strings in `VERSION` is fine as far as I can tell. I could be wrong. I was talking about `SOVERSION` which is different

**Comment by ghisvail** [comment]
> @shehzan10 having strings in VERSION is fine as far as I can tell.

Indeed.

> Can you use FG_VERSION_MAJOR here for consistency ?

I left it at 0 for now to stay on the safe side (i.e. no stable API).

Feel free to give it a different number when ready.

**Pull request merged**